internal/cmd/controller/cleanup/root.go

```diff
@@ -0,0 +1,40 @@
+package cleanup
+
+import (
+	"fmt"
+
+	command "github.com/rancher/fleet/internal/cmd"
+	"github.com/rancher/fleet/pkg/version"
+	"github.com/spf13/cobra"
+)
+
+var (
+	debugConfig command.DebugConfig
```

Comment:
This should not be needed, since the root command already configures it

Reply:
That's because we merged something else first, this PR is in conflict.

Reply:
@manno is right. It has conflicts with https://github.com/rancher/fleet/pull/1899 which is the PR that changed the debug approach. I'm fixing the conflicts